components/swagRock/CardAnyZap.tsx

```diff
@@ -332,8 +335,17 @@ function CardAnyZap(): ReactElement {
 					</div>
 				</div>
 
-				<div aria-label={'card actions'}>
-					<div className={'mb-3'}>{renderButton()}</div>
+				<div aria-label={'card actions'} className={'mb-3'}>
+					<div>{renderButton()}</div>
+					<span className={'flex flex-row items-center justify-center'}>
+						<a
+							className={'cursor-pointer pt-2 text-center text-xs text-neutral-400/60 transition-colors hover:text-neutral-400'}
+							href={'https://docs.joinwido.com/'}
```

Comment:
Is this the link they want? I would think they'd like the homepage to be linked here
```suggestion
							href={'https://www.joinwido.com/'}
```